src/main/java/greencity/entity/Place.java

```diff
@@ -37,7 +37,7 @@
     @Column(unique = true, length = 50)
     private String email;
 
-    @OneToMany(mappedBy = "place", fetch = FetchType.LAZY, cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @OneToMany(mappedBy = "place", cascade = {CascadeType.PERSIST, CascadeType.MERGE})
```

Comment:
Why you delete FetchType.LAZY? You will get all comments every time, when you will get place from database.